packages/v4/src/content/design-guidelines/content/HTTP-404-pages.md

```diff
@@ -0,0 +1,100 @@
+---
+id: HTTP 404 pages
+section: UX writing
+---
+
+A 404 page is an error page that a user lands on when their requested content doesn’t exist or can’t be found. 404 pages are named after the type of error they communicate: “Error 404: Not found.”
+
+Write 404 pages with [error message best practices](/ux-writing/error-messages) in mind: Explain what a 404 error is, define how users can proceed, and provide the tools they need to get there.
+
+Effective 404 pages combine several elements to regroup, redirect, and empower lost users to reach their desired destination or find a new one.
+
+<img src="./img/404-page.png" alt="The PatternFly 404 page with added call outs to mark four main elements: Heading, next steps, suggested content, and a link to the home page." />
+
+1. **Heading:** Communicates what a 404 error is in plain language that users can understand. Avoid including “404” or “404 error” unless it’s clearly defined after: “404: That page no longer exists.”
+
+2. **Next steps:** Define how users can proceed, typically by inviting them to search the site, explore suggested content, or both.
+
+3. **Suggested content (optional):** Points users toward relevant pages such as onboarding information, reference guides, or FAQs.
+
+4. **Link to home page:** Provides a quick and easy way for users to navigate back to your site’s home page.
+
+## Writing 404 page content
+Consider your 404 page as a flight path instead of a dead end. Use clear, informative microcopy to point users in the right direction.
+
+**Use understandable language.** Skip technical jargon by writing your 404 error message in plain and specific terms.
+
+<div class="ws-content-table">
+
+| **Don't**                      | **Do**                  |
+|:-------------------------------:|:--------------------------:|
+| Error 404: Not found | 404: THat page no longer exists| 
```

Comment:
Oops, jut noticed a typo, should be "That" not "THat"

Reply:
Yikes! I'll edit that out — good catch.